cimlab/loaders/gridappsd/__init__.py

```diff
@@ -82,22 +85,22 @@ def get_logger(self):
     def query_data(self, query, database_type="powergridmodel", timeout=30):
         return self.__gapps__.query_data(query, database_type, timeout)
 
-    def create_default_instances(self, feeder_mrid: str | cim.Feeder, mrid_list: List[str]) -> List[object]:
+    def create_default_instances(self, feeder_mrid: str | self.cim.Feeder, mrid_list: List[str]) -> List[object]:
```

Comment:
self.cim.feeder doesn't make sense in this context.  it should be cim.Feeder

The reason it's not liking it is because it's dynamically loaded.  so you can specify that it only runs the typing hints at compile time